src/flows/context/flow.list.tsx

```diff
@@ -256,9 +253,10 @@ export const FlowListProvider: FC = ({children}) => {
       name: flow.name,
       range: flow.range,
       refresh: flow.refresh,
-      data: flow.data.serialize ? flow.data.serialize() : flow.data,
-      meta: flow.meta.serialize ? flow.meta.serialize() : flow.meta,
+      data: flow.data,
+      meta: flow.meta,
```

Comment:
FYI the new library seems to have solved this issue since it is strongly typed https://github.com/influxdata/ui/pull/567#discussion_r566996414